keymap/tutorial.kbd

```diff
@@ -826,11 +826,11 @@
   So, again with the mini-language, we set foo to:
     (tap-hold 200 x lsft) ;; Like tap-next, but with a 200ms timeout
   Then:
-    Tesc            -> x
-    Tesc Ta         -> xa
-    Pesc 300 a      -> A (the moment you press a)
-    Pesc a 300      -> A (after 200 ms)
-    Pesc a 100 Resc -> xa (both happening immediately on Resc)
+    Tesc             -> x
+    Tesc Ta          -> xa
```

Comment:
You could have removed a space in the changed lines and would have gotten a smaller diff.

Reply:
These spaces were added to preserve aligment of the arrows in this block of examples. It looks better to me. Is it relevant the size of the diff?

Reply:
In general diff size is not that relevant just a good boni.
What I meant is that you could have removed a space in the lines you did edit since you would keep alignment but also have a small diff.
Though just my inner perfectionist speaking.
Adding the extra work of ammending the commit and force pushing, is simply not necessary.